db/imgcodec/webp/create_every_orientation.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
please add the NVIDIA copyright header here

Reply:
It's there, isn't it?

Reply:
It wasn't when I wrote that :)